classes/gateways/class.pmprogateway_stripe.php

```diff
@@ -3956,7 +3956,18 @@ public static function get_application_fee_percentage() {
 		if ( self::using_legacy_keys() ) {
 			return 0;
 		}
-		$application_fee_percentage = pmpro_license_isValid( null, pmpro_license_get_premium_types() ) ? 0 : 1;
+
+		//These are percentage values 0%, 1% and 2%
+		
+		$stripe_app_fee = pmpro_getOption( 'stripe_app_fee' );
+		if ( $stripe_app_fee === FALSE ) {
```

Comment:
```suggestion
		if ( ! empty( $stripe_app_fee ) ) {
```
We should use the saved application fee if there is one saved in options.